assets/src/components/detours/pastDetourPanel.tsx

```diff
@@ -1,12 +1,87 @@
 import React from "react"
 import { Panel } from "./diversionPage"
+import { DetourDirection } from "../../models/detour"
+import { Stop } from "../../schedule"
+import { ArrowLeft } from "../../helpers/bsIcons"
+import { Button, ListGroup } from "react-bootstrap"
+import { AffectedRoute, MissedStops } from "./detourPanelComponents"
 
-export const PastDetourPanel = () => (
+export interface PastDetourPanelProps {
+  directions?: DetourDirection[]
+  connectionPoints?: string[]
+  missedStops?: Stop[]
+  routeName: string
+  routeDescription: string
+  routeOrigin: string
+  routeDirection: string
+  onNavigateBack: () => void
+}
+
+export const PastDetourPanel = ({
+  directions,
+  connectionPoints,
+  missedStops,
+  routeName,
+  routeDescription,
+  routeOrigin,
+  routeDirection,
+  onNavigateBack,
+}: PastDetourPanelProps) => (
   <Panel as="article">
     <Panel.Header className="">
-      <h1 className="c-diversion-panel__h1 my-3">Past Detour</h1>
+      <h1 className="c-diversion-panel__h1 my-3">View Past Detour</h1>
     </Panel.Header>
 
-    <Panel.Body className="d-flex flex-column"></Panel.Body>
+    <Panel.Body className="d-flex flex-column">
+      <Panel.Body.ScrollArea>
+        <div className="d-flex justify-content-between align-items-center">
+          {onNavigateBack && (
+            <Button
+              className="icon-link my-3"
+              variant="outline-primary"
+              onClick={onNavigateBack}
+            >
+              <ArrowLeft />
+              Back
+            </Button>
+          )}
+        </div>
+        <AffectedRoute
+          routeName={routeName}
+          routeDescription={routeDescription}
+          routeOrigin={routeOrigin}
+          routeDirection={routeDirection}
+        />
+
+        {connectionPoints && (
+          <section className="pb-3">
+            <h2 className="c-diversion-panel__h2">Connection Points</h2>
+            <ListGroup as="ul">
```

Comment:
note: I do find it annoying that the bootstrap react library doesn't default to `ul` >.>

Reply:
I guess, I mean that I don't like that it defaults to a `div`, given it could be an `ol`